x-pack/plugins/aiops/server/routes/queries/fetch_index_info.ts

```diff
@@ -61,18 +70,29 @@ export const fetchIndexInfo = async (
     { signal: abortSignal, maxRetries: 0 }
   );
 
+  const allFieldNames: string[] = [];
+
   const finalFieldCandidates: Set<string> = new Set([]);
+  const finalTextFieldCandidates: Set<string> = new Set([]);
   const acceptableFields: Set<string> = new Set();
+  const acceptableTextFields: Set<string> = new Set();
 
   Object.entries(respMapping.fields).forEach(([key, value]) => {
     const fieldTypes = Object.keys(value) as ES_FIELD_TYPES[];
     const isSupportedType = fieldTypes.some((type) => SUPPORTED_ES_FIELD_TYPES.includes(type));
     const isAggregatable = fieldTypes.some((type) => value[type].aggregatable);
+    const isTextField = fieldTypes.some((type) => SUPPORTED_ES_FIELD_TYPES_TEXT.includes(type));
 
     // Check if fieldName is something we can aggregate on
     if (isSupportedType && isAggregatable) {
       acceptableFields.add(key);
     }
+
+    if (isTextField) {
```

Comment:
In other places (e.g. advanced job wizard) we allow the user to inspect other field values for `text` fields. Is it not possible to allow this here too?

<img width="211" alt="image" src="https://github.com/elastic/kibana/assets/7405507/95e44691-89ec-4e54-b420-c0dc73a2590c">

Job wizard:
<img width="272" alt="image" src="https://github.com/elastic/kibana/assets/7405507/4ba26ee6-365f-42b5-8bea-2fa99d9ef9a5">

If not, could you add some padding to the left so that the text aligns with other field types?